packages/fluentui/react-telemetry/src/TelemetryPerfFlags.tsx

```diff
@@ -10,7 +10,7 @@ type TelemetryPerfFlagsProps = {
 
 export const TelemetryPerfFlags: React.FC<TelemetryPerfFlagsProps> = props => (
   <div style={styles.performanceFlags()}>
-    {Object.keys(props.flags).map(flag => (
+    {Object.keys(defaultPerformanceFlags).map(flag => (
```

Comment:
It's required to iterate on `defaultPerformanceFlags` as currently we're reading them `localStorage`: it will be an issue if we will introduce a new perf flag.